tests/unit/clients/test_auth.py

```diff
@@ -252,6 +273,21 @@ async def test_refresh_token_with_app_default_cred(client_with_app_default_cred,
     assert 2 == len(caplog.records)
 
 
+@pytest.mark.asyncio
+async def test_refresh_token_with_compute_engine_cred(
+        client_with_compute_engine_cred, mock_parse_expiry,
+        caplog, payload_resp_refresh_token):
+    """Successfully refresh access token with compute engine credentials ."""
+    url = "http://metadata.google.internal/" \
+          "computeMetadata/v1/instance/service-accounts/default/token"
```

Comment:
Also quotes here, as well as parentheses for the string concatenation

Reply:
yep will do